plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java

```diff
@@ -270,7 +270,7 @@ public CassandraTable getTable(SchemaTableName schemaTableName)
                 .sorted(comparing(CassandraColumnHandle::getOrdinalPosition))
                 .collect(toList());
 
-        CassandraTableHandle tableHandle = new CassandraTableHandle(tableMeta.getKeyspace().asInternal(), tableMeta.getName().asInternal());
+        CassandraTableHandle tableHandle = new CassandraTableHandle(new CassandraNamedRelationHandle(tableMeta.getKeyspace().asInternal(), tableMeta.getName().asInternal()));
```

Comment:
What is your opinion on returning as `CassandraNamedRelationHandle` ?

Reply:
Could you elaborate on that?

Reply:
Currently `CassandraTable` has two fields - `CassandraTableHandle` and `List<CassandraColumnHandle`. `CassandraTableHandle` is wrapper for both named and query based relation - which might be an overkill here. Instead can we have something like this 
`public CassandraTable(CassandraNamedRelationHandle tableHandle, List<CassandraColumnHandle> columns)`

Just thinking out aloud.